docs-md/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -186,6 +189,15 @@ The square root of a value.
 Collections
 ===========
 
+ARRAY_LENGTH
+------------
+
+`ARRAY_LENGTH(ARRAY[1, 2, 3])`
+
+Given an array, return the number of elements in the array.
+
+If the supplied parameter is null the method returns 0.
+

```

Comment:
sorry, i realize i should have seen this `array_length` function going in separately but adding the docs here has called it to my attention :)   - not sure we should return zero for a null input array, that conflicts with the result for an empty array. I would expect that `array_length(null)` returns `NULL`.  From what i can see in a quick google around, that looks to be more common but it's not well documented in many places.

Reply:
>  not sure we should return zero for a null input array, 

I was mulling this over myself.  As you've raised it to... consider it done!

Reply:
Pull the ARRAY_LENGTH stuff out into different PR now that it's growing:

https://github.com/confluentinc/ksql/pull/4848